debug_toolbar/panels/staticfiles.py

```diff
@@ -29,7 +31,9 @@ def url(self):
 
 
 # This will collect the StaticFile instances across threads.
-used_static_files = ContextVar("djdt_static_used_static_files")
+used_static_files = ContextVar("djdt_static_used_static_files", default=[])
```

Comment:
```suggestion
used_static_files = ContextVar("djdt_static_used_static_files", default=list)
```
Do we run the same risk here as we would with defining a default on a method argument of `[]` versus `None` and instantiating the list?